### ISSUE TITLE
[Enhancement] Pilots and command points cap increase/removal

### ISSUE THREAD

**Issue by NerdyQT** [PavelZinchenko/event-horizon-frontier-builds#23]
- Remove the cap of pilots that can be owned.
- Remove or increase the cap of command points so more ships can be used.

**Comment by PavelZinchenko** [comment]
Can't do that. It'd kill the game balance

**Comment by cktgh** [comment]
I agree that command points definitely has to be limited, but removing/increasing the cap of pilots is does not affect game balance, and it creates extra goals for end game players.

**Comment by PavelZinchenko** [comment]
It'll allow players to have an army of frigates in the arena. Increased it to 20 for now. Might increase it more later.

**Comment by cktgh** [comment]
Right now we could surpass the pilot limit by obtaining pilots from exploration.
I suppose an army of frigates isn't practical (or even destroyers for that matter), due to the small number of good frigates.

**Comment by TaipionsSalemClient** [comment]
> It'll allow players to have an army of frigates in the arena. Increased it to 20 for now. Might increase it more later.

There are only a handful or useful frigates in the game, it would not hurt to increase the pilot cap to 30
